test/SeedData/populateDb.test.ts

```diff
@@ -0,0 +1,227 @@
+import * as sinon from 'sinon';
+import BigNumber from 'bignumber.js';
+import { interacts } from '@openst/mosaic-contracts';
+import SeedData from '../../src/SeedData';
+import { Config } from '../../src/Config/Config';
+import Repositories from '../../src/repositories/Repositories';
+import AuxiliaryChain from '../../src/models/AuxiliaryChain';
+import Gateway from '../../src/models/Gateway';
+import { GatewayType } from '../../src/repositories/GatewayRepository';
+import AuxiliaryChainRepositoryUtil from '../repositories/AuxiliaryChainRepository/util';
+import GatewayRepositoryUtil from '../repositories/GatewayRepository/util';
+import ContractEntityRepositoryUtil from '../repositories/ContractEntityRepository/util';
+import ContractEntity, { EntityType } from '../../src/models/ContractEntity';
+
+const Web3 = require('web3');
+
+describe('SeedData.populateDb()', () => {
+  let config: Config; let seedData: SeedData; let
+    repositories: Repositories;
+  let web3: any;
+
+  const originChain = '12346';
+  const auxiliaryChainId = 301;
+  const zeroBn = new BigNumber('0');
+  const ostComposerAddress = '0x3c8ba8caecb60c67d69605a772ae1bb9a732fb38';
+  const ostGatewayAddress = '0x97BA58DBE58898F2B669C56496f46F638DC322d4';
+  const ostCoGatewayAddress = '0x40ce8B8EDEb678ea3aD1c9628924C903f8d04227';
+  const anchorAddress = '0xaC80704c80AB83512b48314bDfa82f79923C2Fbe';
+  const coAnchorAddress = '0xBe26124167E8a350eE806B3ba11Ddb6c8E6dc689';
+  const simpleTokenAddress = '0x325f05a75999347b7d8461BaEf274afAE0B8AE1c';
+  const ostPrimeAddress = '0x0d3E57044B1B96a257fB2ba3958c1130219A2d55';
+
+  async function verifyDataInAuxiliaryChainsTable() {
```

Comment:
Please add documentation for these function.
Same comment for other functions also.